### PR TITLE
Update LABELS.md

### PR DESCRIPTION
**Pull request by oorjitchowdhary**
Fixes #0000 (<=== Add issue number here)
@publiclab/mentors 
Added new labels and updated LABELS.md
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: doc/LABELS.md [hunk 1]
```diff
@@ -9,12 +9,19 @@ Before contributing to the organization, one should be aware of these and keep t
...
+* ` more-detail-please ` tells the issue lacks proper description and perhaps needs code links or the location of the problem.
 * Labels like ` HTML ` , ` CSS ` , ` Ruby ` and ` JavaScript ` tell the **programming language** of the issue.
 * ` design ` - This says that the issue requires more design work and discussion (i.e. mockups and sketches).
+* ` documentation ` - This tells that a certain feature lacks proper documentation or needs more documents.
+* ` testing ` - These issues are usually for adding `unit tests`, `integration tests` or any other tests for a particular feature/program.
 * ` outreach ` - The outreach issues involve community involvement and helping people who're stuck somewhere.
-* Some issues have been labeled with ` fto-candidate ` , ` gci-candidate ` and ` rgsoc ` which mean that these issues have been reserved for students who're participating in these events.
+* Some issues have been labeled with ` summer-of-code ` , ` outreachy ` , ` gci-candidate ` and ` rgsoc ` which mean that these issues have been reserved for students who're participating in these events.
```
Comment (SidharthBansal) [inline doc/LABELS.md]: Please include first timers only also. They are reserved for first timer collaborators only

Comment (oorjitchowdhary) [reply doc/LABELS.md]: I included `first-timers-only` as a new line above.. I actually have described this line as for people participating in various events

**Comment by SidharthBansal** [comment]
I can understand your point.
First timers are also compulsory for these students if different program. So we should include it

**Comment by oorjitchowdhary** [comment]
Okay, I'll add it

**Comment by oorjitchowdhary** [comment]
Done

**Comment by SidharthBansal** [comment]
Thanks for the great work you are doing here at Public Lab.

**Pull request merged**